src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8394,11 +8420,13 @@ private void VerifyQuotedInvalid<T>(string name, string value, T expected, Func<
             [Fact]
```

Comment:
This test does validate that on Linux/Mac we use Portable PDB and otherwise Windows PDB. ConditionalFact does the same check of ```PathUtilities.IsUnixLikePlatform``` value as the line below.